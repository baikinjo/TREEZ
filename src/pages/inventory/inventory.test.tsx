import React from 'react'
import { shallow } from 'enzyme'

import InventoryPage from './inventory'
import InventoryCard from '../../components/inventory/inventory.card'

describe('Inventory Page', () => {
  let wrapper: any
  let mockItems = [{ id: 1 }]
  beforeEach(() => {
    const mockCollection = {
      items: mockItems,
      title: 'Test'
    }

    wrapper = shallow(<InventoryPage />)
  })

  it('should render the InventoryPage component', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should render the same number of Inventory Card as collection array', () => {
    expect(wrapper.find(InventoryCard).length).toBe(mockItems.length)
  })
})
