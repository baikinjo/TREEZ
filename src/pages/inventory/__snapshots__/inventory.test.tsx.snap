// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inventory Page should render the InventoryPage component 1`] = `ShallowWrapper {}`;
